container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -585,11 +585,13 @@ impl pallet_cc_authorities_noting::Config for Runtime {
 
 // To match ethereum expectations
 const BLOCK_GAS_LIMIT: u64 = 15_000_000;
+const MAX_POV_SIZE: u64 = 5 * 1024 * 1024;
 
 impl pallet_evm_chain_id::Config for Runtime {}
 
 parameter_types! {
     pub BlockGasLimit: U256 = U256::from(BLOCK_GAS_LIMIT);
+    pub const GasLimitPovSizeRatio: u64 = BLOCK_GAS_LIMIT.saturating_div(MAX_POV_SIZE);
```

Comment:
```suggestion
    pub GasLimitPovSizeRatio: u64 = BLOCK_GAS_LIMIT.saturating_div(NORMAL_DISPATCH_RATIO * MAX_POV_SIZE);

```

Reply:
This should be     pub GasLimitPovSizeRatio: u64 = BLOCK_GAS_LIMIT.saturating_div(NORMAL_DISPATCH_RATIO * MAX_POV_SIZE), but it would round down to 3 unless we tell it to round up

Reply:
So we'll set it to `4`, right? What is wrong with its rounding down to `3`?